code/game/machinery/suit_storage_unit.dm

```diff
@@ -312,6 +312,8 @@
 	if(shocked)
 		if(shock(user, 100))
 			return
+	if (user.a_intent != INTENT_HELP)
+		return ..()
 	if(!is_operational())
 		if(panel_open)
 			to_chat(usr, "<span class='warning'>Close the maintenance panel first.</span>")
```

Comment:
This might work better under the `if(!is_operational())` check instead, that way it works the same as normal when it's powered, and checks the attack intent when it's unpowered.
Currently it only allows you to interact with it while you're in help intent, which *does* work, but it could cause some confusion:

https://user-images.githubusercontent.com/57483089/135884335-c86c8b15-125c-498a-89a2-69d0140cabcd.mp4

```suggestion
	if(!is_operational())
		if(user.a_intent != INTENT_HELP)
			return ..()
		if(panel_open)
			to_chat(usr, "<span class='warning'>Close the maintenance panel first.</span>")
```

Reply:
Gotcha. I was trying to figure out if there was a typical "standard" of sorts for how interactions between intents should work, and seeing its current behavior, that makes a lot of sense